universal-login-react/src/core/utils/getMinimalAmountForFiatProvider.ts

```diff
@@ -13,16 +13,25 @@ export const getMinimalAmountForFiatProvider = async (
   switch (paymentMethod) {
     case TopUpProvider.RAMP: {
       const currencyPriceInEth = await tokenPricesService.getTokenPriceInEth(currencyDetails);
-      const providerMinimalAmountInFiat = '1';
-      const etherPriceInGBP = (await tokenPricesService.getEtherPriceInCurrency('GBP')).toString();
+      const providerMinimalAmountInFiat = '0.6';
+      const etherPriceInGBP = (await tokenPricesService.getEtherPriceInCurrency('EUR')).toString();
       const providerMinimalAmount = getPriceInEther(providerMinimalAmountInFiat, etherPriceInGBP);
       const providerMinimalAmountInToken = safeDivide(providerMinimalAmount, currencyPriceInEth);
+      const providerMinimalAmountInFiatForRevolut = '2';
+      const providerMinimalAmountForRevolut = getPriceInEther(providerMinimalAmountInFiatForRevolut, etherPriceInGBP);
+      const providerMinimalAmountForRevolutInToken = safeDivide(providerMinimalAmountForRevolut, currencyPriceInEth);
       const requiredDeploymentBalanceAsBigNumber = utils.parseEther(requiredDeploymentBalance);
-      const biggerAmount = bigNumberMax(
-        requiredDeploymentBalanceAsBigNumber,
-        providerMinimalAmountInToken,
-      );
-      return ValueRounder.ceil(utils.formatEther(biggerAmount));
+      const biggerAmounts = [
+        bigNumberMax(
+          requiredDeploymentBalanceAsBigNumber,
+          providerMinimalAmountInToken,
+        ),
+        bigNumberMax(
+          requiredDeploymentBalanceAsBigNumber,
+          providerMinimalAmountForRevolutInToken,
+        ),
+      ];
+      return [ValueRounder.ceil(utils.formatEther(biggerAmounts[0])), ValueRounder.ceil(utils.formatEther(biggerAmounts[1]))];
```

Comment:
return object: 
```ts
{
    minimalAmountForRevolut: ...,
    generalMinimalAmount: ...,
}
```
We should avoid returning an array of values because it is very easy to make a mistake handling it, especially when these values are semantically two different things e.g. returning an array of tokens is correct.